agents/windows/plugins/win_license.bat

```diff
@@ -5,4 +5,13 @@ REM * plugin to gather and output Windows activation status
 REM ***
 
 echo ^<^<^<win_license^>^>^>
-cscript //NoLogo %windir%/System32/slmgr.vbs /dli
+if not exist "%temp%\slmgr.vbs" (
```

Comment:
Can we change the name of the copied script to `checkmk_slmgr.vbs`? Unlikely that naming clashes exist, but I'm rather more defensive here.